spec/requests/api/v1/projects/field_sets_spec.rb

```diff
@@ -0,0 +1,177 @@
+require 'rails_helper'
+
+RSpec.describe 'Field Sets Requests', type: :request do
```

Comment:
Another case where we'll need to add include_examples 'requires user to have correct permissions' do and check for project admin permissions, but that can come later too. Probably want to add a TODO here as well.

Reply:
Project permissions have to no been implemented. I am working on that right now and I'll add tests to all these routes at that point.